app/config/question-bank.js

```diff
@@ -2939,7 +2948,7 @@ const questionBank = {
             building projects is more than £15,000 (40% of £37,500), you may still be eligible to apply for grant funding.
             `,
             insertText: '',
-            additionalPara: `If you are applying for grant funding for a single building, you can <a href="https://www.gov.uk/government/organisations/rural-payments-agency"> see other grants you may be eligible for.</a>`,
+            additionalPara: `If you are applying for grant funding for a single building, you can <a href="https://www.gov.uk/government/organisations/rural-payments-agency"> See other grants you may be eligible for</a>`,
```

Comment:
Why is the see capitalised? Looks like it isnt in line 2803 so not sure why it is here

Reply:
@zavies97  good point, will revert it.

Reply:
Should this instead be **<a href="https://www.gov.uk/government/organisations/rural-payments-agency">see** (as this was the change made to the other additionalPara)?

Reply:
done